app/controllers/api/v2/events_controller.rb

```diff
@@ -2,7 +2,10 @@ class Api::V2::EventsController < Api::BaseController
 
   def create
     event = JSON.parse(request.body.read)
-    ap event
+    push_events = event['events'].find_all do |e|
```

Comment:
I would suggest to build Event instance from params and then in it after init - perform all the logic which is here right now. So controller will be fatfree:

```rails
Event.new(JSON.parse(request.body.read))
head status: :accepted
```